_posts/2016-01-21-Rust-1.6.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "Announcing Rust 1.6"
+author: The Rust Core Team
+---
+
+Hello 2016! We’re happy to announce the first Rust release of the year, 1.6.
+Rust is a systems programming language focused on safety, speed, and
+concurrency.
+
+As always, you can [install Rust 1.6][install] from the appropriate page on our
+website, and check out the [detailed release notes for 1.6][notes] on GitHub.
+About 1100 patches were landed in this release.
+
+[install]: http://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/stable/RELEASES.md#version-160-2016-01-21
+
+### What's in 1.6 stable
+
+This release contains a number of small refinements, one major feature, and
+a change to [Crates.io](https://crates.io).
+
+#### libcore stabilization
+
+The largest new feature in 1.6 is that [`libcore`] is now stable! Rust’s standard
+library is two-tiered: `libcore` contains a lot of functionality, but more important
```

Comment:
I think you want to say: "... is two-tiered: there's a small core library, `libcore`, and the full standard library, `libstd`, that builds on top of it`."